runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/wire/WireCoders.java

```diff
@@ -51,9 +49,9 @@
    *
```

Comment:
Fix comment like above to say we modify the components builder and also update what we return

Reply:
Please fix the comment since I still think this applies.

Reply:
Not sure what you mean: I added a paragraph to the comment saying this adds coders to the builder, and modified `@return` to say it returns the id of the new coder.